build.gradle

```diff
@@ -1,18 +1,18 @@
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.2'
+    gradleVersion = '3.5'
 }
 
 allprojects {
-    version = "2.0.13"
+    version = "2.0.15"
 }
 
 buildscript {
     repositories {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
-        classpath 'com.novoda:bintray-release:0.4.0'
+        classpath 'com.android.tools.build:gradle:2.3.1'
+        classpath 'com.novoda:bintray-release:0.5.0'
```

Comment:
Updated all these dependencies so that we can have a composite build in another project, to avoid shipping this halfway house code 😄